ops/create-cluster.sh

```diff
@@ -42,7 +42,7 @@ readonly max_retry_count=50
 retries=0
 
 while [[ -z ${cassandra_host} || -z ${DEIS_ROUTER_HOST_ROOT} ]]; do
-   cassandra_host=$(kubectl --namespace=cassandra get svc cassandra-cluster-cassan-ext -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
+   cassandra_host=$(kubectl --namespace=cassandra get svc cassandra-cluster-cassan-ext -o jsonpath='{.spec.clusterIP}')
```

Comment:
When I run this from my machine and then plug the ip into cqlsh, I don't manage to connect. Is this expected? I.e., this is a change so that we can only connect to Cassandra from within the cluster?